modules/wrapper/src/main/java/com/beust/kobalt/wrapper/Main.java

```diff
@@ -425,6 +432,18 @@ private void downloadImpl(File file, String version) throws IOException {
         }
     }
 
+    private Proxy getProxy() {
+        String http_proxy = System.getenv("http_proxy");
```

Comment:
Ah ok, so that's how you extract the proxy info, I was wondering what approach you picked.

I was thinking that information should be picked from `~/.config/kobalt/settings.xml` to be consistent, but your approach is better than nothing for now.